platform/persistence/platform-persistence-core-api/src/main/java/org/codice/ddf/persistence/PersistentStore.java

```diff
@@ -57,7 +57,7 @@
      * @return
      * @throws PersistenceException
      */
-    public List<Map<String, Object>> get(String type) throws PersistenceException;
+    public List get(String type) throws PersistenceException;
```

Comment:
Yes, I imagine you didn't want to change the API here unless there was an explicit reason to do so.